src/main/java/Task.java

```diff
@@ -0,0 +1,64 @@
+/**
+ * This represents the superclass Task in each element of Task[] list in Duke.java.
+ */
+public class Task {
+
+    protected String description;
+    protected boolean isDone;
+    public final static String FILE_STRING_SEPARATOR = " | ";
+
+
```

Comment:
Consider removing extra lines in your code so that the formatting is consistent and compact.